ovs_performance.py

```diff
@@ -1818,9 +1826,9 @@ def get_of_port_packet_stats(of_port, **kwargs):
         sys.exit(-1)
 
     slogger.debug("OF port {0} stats: tx = {1}, tx_drop = {2}, rx = {3}, tx_drop = {3}".
-                  format(of_port, long(tx), long(tx_drop), long(rx), long(rx_drop)))
+                  format(of_port, int(tx), int(tx_drop), int(rx), int(rx_drop)))
```

Comment:
Maybe when typecasting twice, we should move the typecast to the assignments above?